src/plugins/plugin.title.js

```diff
@@ -111,23 +111,20 @@ var Title = Element.extend({
 	fit: function() {
 		var me = this;
 		var opts = me.options;
-		var display = opts.display;
-		var minSize = me.minSize;
-		var lineCount = helpers.isArray(opts.text) ? opts.text.length : 1;
-		var fontOpts = helpers.options._parseFont(opts);
-		var textSize = display ? (lineCount * fontOpts.lineHeight) + (opts.padding * 2) : 0;
+		var minSize = me.minSize = {};
```

Comment:
Just want to confirm that re-initializing `minSize` here causes no problems

Reply:
`minSize` is set to new values in all cases, so this won't have any effect. It's also probably not really necessary since it won't have any effect. I'm not sure why I did this and can remove it if you'd prefer